docs/maui/views/popup/popup-result.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Popup<T> - .NET MAUI Community Toolkit
+author: bijington
+description: Popup<T> enables developers to return a value from a popup when it is dismissed.
+ms.date: 05/19/2025
+---
+
+# Returning a value from a Popup
+
+The .NET MAUI Community Toolkit provides the ability to show a [Popup](../Popup.md) to a user. A common more complex scenario is to display a `Popup` and await for a result to be returned when that `Popup` is dismissed. This page covers how the `Popup<T>` class can be used to achieve the desired behavior.
+
+## Building a Popup
+
+A `Popup<T>` can be created in XAML or C# as follows:
+
+### Building a Popup in XAML
+
+The following section covers how to create a `Popup<T>` using XAML.
+
+#### Defining your Popup
+
+The easiest way to create a `Popup<T>` is to add a new `.NET MAUI ContentView (XAML)` to your project, this will create 2 files; a _*.xaml_ file and a _*.xaml.cs_ file. The contents of each file can be replaced with the following:
+
+##### .XAML
```

Comment:
Using `.xaml` in a markdown heading is a bit jarring. Let's just call this section `XAML File`

```suggestion
##### XAML File
```